### PR TITLE
fix(manifest): Correct source of manifest

### LINKED ISSUE #5546: [Feature] prompt bucket install
when trying to install a package from a known bucket (`scoop install extras/eclipse-java`), I think it would be better if scoop prompted the user to add the bucket (`extras ` in this case) instead of saying the manifest couldn't be found (because the user didn't add the bucket beforehand).

### PR DESCRIPTION
**Pull request by HUMORCE**
#### Description

Correct prompt about source of manifest when not found.

#### Motivation and Context

Close #5546

This is the original functionality that was broken when merging new features/refactoring. After the changes, user must explicitly input full filename for local manifests:
```
$manifest = '{
    "version": "1",
    "url": "http://example.org/#/App.exe"
}'

$manifest | out-file localmanifest.json -Encoding ascii
$manifest | out-file manifest_without_suffix -Encoding ascii

scoop install localmanifest × # This was allowed before.
scoop install localmanifest.json √
scoop install .\localmanifest.json √
scoop install manifest_without_suffix √
```

Example for actual usage:
```
~\Desktop\manifests
❯ ls -name
git
git.json
~\Desktop\manifests
❯ scoop info git

Name        : git
Description : Distributed version control system
Version     : 2.42.0.2
Bucket      : main
Website     : https://gitforwindows.org
License     : GPL-2.0-only
Updated at  : 8/30/2023 12:30:31 PM
Updated by  : github-actions[bot]
Installed   : 2.42.0.2
Binaries    : bin\sh.exe | bin\bash.exe | cmd\git.exe | cmd\gitk.exe | cmd\git-gui.exe | cmd\scalar.exe | usr\bin\tig.exe |
              git-bash.exe
Shortcuts   : Git Bash | Git GUI
Environment : GIT_INSTALL_ROOT = <root>
Notes       : Set Git Credential Manager Core by running: "git config --global credential.helper manager"

              To add context menu entries, run '<root>\install-context.reg'

              To create file-associations for .git* and .sh files, run '<root>\install-file-associations.reg'


~\Desktop\manifests

# install git from main bucket
❯ scoop install git

# install git from ~\Desktop\manifests\git file
❯ scoop install .\git

# install git from ~\Desktop\manifests\git.json file, unless a manifest with filename `git.json.json` in added buckets
❯ scoop install git.json

# install git from ~\Desktop\manifests\git.json file
❯ scoop install .\git.json
```

#### How Has This Been Tested?

test.ps1: https://gist.github.com/HUMORCE/4e215c95bdcbb9d17a6b0cda777b6331/raw/790ee7b5b050dfb511ce59e6c83efdd4404292f0/test.ps1

screenshot: left=before, right=after
Screenshot_230711021447

show source for remote or local manifest when installing:
Screenshot_231008140545

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have ensured that I am targeting the `develop` branch.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.
- [x] I have added an entry in the CHANGELOG.

### CODE REVIEW

File: lib/manifest.ps1 [hunk 1]
```diff
@@ -44,24 +44,23 @@ function Get-Manifest($app) {
         if ($bucket) {
             $manifest = manifest $app $bucket
         } else {
-            foreach ($bucket in Get-LocalBucket) {
-                $manifest = manifest $app $bucket
+            foreach ($tekcub in Get-LocalBucket) {
```
Comment (niheaven) [inline lib/manifest.ps1:47]: The `$tekcub` is? A random name?

Comment (HUMORCE) [reply lib/manifest.ps1:47]: no, it's simply spelled backward of `bucket`.

**Review by niheaven — DISMISSED** [review]

**Review by niheaven — DISMISSED** [review]

File: lib/depends.ps1 [hunk 2]
```diff
@@ -37,9 +37,9 @@ function Get-Dependency {
 
         if (!$manifest) {
             if (((Get-LocalBucket) -notcontains $bucket) -and $bucket) {
-                warn "Bucket '$bucket' not installed. Add it with 'scoop bucket add $bucket' or 'scoop bucket add $bucket <repo>'."
+                warn "Bucket '$bucket' not Added. Add it with $(if($bucket -in (known_buckets)) { "'scoop bucket add $bucket' or " })'scoop bucket add $bucket <repo>'."
```
Comment (HUMORCE) [inline lib/depends.ps1:40]:
```suggestion
                warn "Bucket '$bucket' not added. Add it with $(if($bucket -in (known_buckets)) { "'scoop bucket add $bucket' or " })'scoop bucket add $bucket <repo>'."
```

Comment (HUMORCE) [reply lib/depends.ps1:40]: Commit this before merge, thanks

**Review by niheaven — APPROVED** [review]

**Pull request merged**